<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>295</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_unsortedFile">
       <property name="text">
        <string>Исходный файл:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_unsortedFile">
       <item>
        <widget class="QLineEdit" name="lineEdit_unsortedFile">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Введите путь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_unsortedFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_sortedFile">
       <property name="text">
        <string>Отсортированный файл:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_sortedFile">
       <item>
        <widget class="QLineEdit" name="lineEdit_sortedFile">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Введите путь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_sortedFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_workFileCount">
       <property name="text">
        <string>Количество рабочих файлов:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_workFileCount"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_sort">
       <property name="text">
        <string>Добро</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="progressBar">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_sort</tabstop>
  <tabstop>lineEdit_unsortedFile</tabstop>
  <tabstop>lineEdit_sortedFile</tabstop>
  <tabstop>spinBox_workFileCount</tabstop>
  <tabstop>toolButton_unsortedFile</tabstop>
  <tabstop>toolButton_sortedFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
